src/app/progress/SeasonalRank.tsx

```diff
@@ -64,6 +64,16 @@ export default function SeasonalRank({
   const { progressToNextLevel, nextLevelAt } = prestigeMode ? prestigeProgress : seasonProgress;
   const { rewardItems } = defs.Progression.get(seasonPassProgressionHash);
 
+  const brightEngramRewardLevelsTemp: number[] = [];
+
+  rewardItems
+    .filter((item) => item.itemHash === brightEngramHash)
+    .forEach((item) => {
+      brightEngramRewardLevelsTemp.push(item.rewardedAtProgressionLevel % 10);
+    });
+
+  const brightEngramRewardLevels = [...new Set(brightEngramRewardLevelsTemp)];
```

Comment:
2 very nitpicky matters that are entirely about taste:
- you could `let` (instead of `const`) the original `brightEngramRewardLevelsTemp` variable, and assign to it again here, if you wanted to avoid extraneous intermediate "temp" var names
- `_.uniq` might serve well instead of `[...new Set(`, to explicitly name+describe the operation you're performing. instead of relying on next reader to recognize the syntax & "trick" happening here

Reply:
totally forgot about `_.uniq`